e2e/components/Banner.test.ts

```diff
@@ -0,0 +1,93 @@
+import {test, expect} from '@playwright/test'
```

Comment:
For my learning... is it fair to say that the majority of things (e.g. keyboard interactions, markup behavior) can be well-tested with jest unit tests? 

I'm curious if there are any plans or any needs to expand e2e testing coverage for Primer react component beyond axe and snapshot tests.

Reply:
@khiga8 great questions 🔥 Thankfully I think with: https://www.npmjs.com/package/@testing-library/user-event we can get pretty robust keyboard interaction testing in Jest.

No plans so far to extend what is in the e2e tests, I think the only other case that has come up is when we need to test behavior when the browser is resizing (like for underline nav).